src/vm/fieldmarshaler.h

```diff
@@ -69,6 +69,48 @@ class FieldMarshaler_Exception;
 class FieldMarshaler_Nullable;
 #endif // FEATURE_COMINTEROP
 
+enum ParseNativeTypeFlags
+{
+    ParseNativeTypeFlag_None = 0x00,
+    ParseNativeTypeFlag_IsAnsi = 0x01,
+
+#ifdef FEATURE_COMINTEROP
+    ParseNativeTypeFlag_IsWinRT = 0x02,
+#endif // FEATURE_COMINTEROP
+};
+
+inline ParseNativeTypeFlags operator|=(ParseNativeTypeFlags& lhs, ParseNativeTypeFlags rhs)
+{
+    LIMITED_METHOD_CONTRACT;
+    lhs = static_cast<ParseNativeTypeFlags>(lhs | rhs);
+    return lhs;
+}
+
+BOOL IsFieldBlittable(FieldMarshaler* pFM);
+
+#ifdef _PREFAST_
```

Comment:
Since you pulled this into a header, all this `#ifdef` nonsense can be removed from the declaration.